web/src/views/DandisetLandingView/DandisetDetails.vue

```diff
@@ -152,10 +186,24 @@ export default {
 
       return null;
     },
+    formattedSize() {
+      const { stats } = this;
+      if (!stats) return undefined;
+
+      const a = filesize(stats.bytes);
+      return a;
```

Comment:
```suggestion
      if (!stats) {
        return undefined;
      }
      return filesize(stats.bytes);
```
Java style guides almost universally ban the one line if statement, so I habitually avoid whenever possible. It's not terribly important to me, but I see no reason for the `a`.

Reply:
Whoops, that was left over from me quickly testing it. Good catch.